mypy/nodes.py

```diff
@@ -2211,6 +2208,18 @@ def get_containing_type_info(self, name: str) -> 'Optional[TypeInfo]':
                 return cls
         return None
 
+    @property
+    def protocol_members(self) -> List[str]:
+        # Protocol members are names of all attributes/methods defined in a protocol
```

Comment:
Should this return an empty list if the class is not a protocol, similar to how this was defined earlier?

Reply:
I would say it is not necessary, but I see no particular danger in this.